jest.config.js

```diff
@@ -2,4 +2,8 @@ module.exports = {
     rootDir: '.',
     setupFilesAfterEnv: ['<rootDir>config/jest/enzymeSetup.js'],
     roots: ['<rootDir>/packages'],
+    testPathIgnorePatterns: [
+        '/node_modules/',
+        'Transfer/__tests__/common/createChildren.js',
```

Comment:
I **do not** want to have the `jest.config.js` change (`testPathIgnorePatterns`) in `master`.

I did a "best effort" merge last night since you mentioned that the Transfer will need some refactoring once it's moved to UI. This should be part of that.